docs/_templates/2018/index.html

```diff
@@ -91,11 +91,22 @@
 
   <div class="about">
     <div class="uk-container">
+
       <div class="about-content">
         <h2>What is Write the Docs about?</h2>
         <p>Write the Docs brings everyone who writes the docs together in the same room: Programmers, Tech Writers, Support, Designers, Developer Advocates, and more. We all have things to learn from each other, and there’s no better way than sitting together and talking.</p>
         <p>We invite you to join {{ about.attendees }} other folks in our event to explore the art and science of documentation.</p>
+
+        {% if cobrand.logo %}
+        <h2>What is Open Help about?</h2>
+        <p>The Open Help Conference features two days of presentations and open discussions. In true open source spirit, our discussions and agenda are led by our attendees, allowing you to learn and share what matters to you most. With some of the most interesting people in the industry, the Open Help Conference showcases the ideas that are shaping the future of help.</p>
+
+        <h2>What is this event?</h2>
+        <p>This event is a combination of the two communities and conferences. We will have a one day Write the Docs style talk track, followed by a combination day of discussions. Then we'll do an Open Help style 3-day sprint.</p>
```

Comment:
"combination day" ??

Reply:
Or maybe "This event combines the two communities and conferences."

Reply:
Maybe revise the other two sentences to:

We will have one day of Write the Docs-style talks, followed by one day of combined Write the Docs and Open Help discussions. Then we'll have an Open Help-style three-day documentation sprint.